### PR TITLE
Block list bugfix

### PR DESCRIPTION
**Pull request by aemerick**
Small bugfix allowing .block_list, .parameter, .file_list, and .libconfig files to be output in data-type outputs when no sub-directory is provided. In this case, the prefix to these files is adopted from the corresponding output file name. 

Fixes Issue #32

### CODE REVIEW

**Comment by bwoshea** [comment]
@brittonsmith I'm adding you as a reviewer.  I will also add myself.

File: src/Cello/io_OutputData.cpp [hunk 1]
```diff
@@ -133,57 +133,65 @@ void OutputData::write_block ( const  Block * block ) throw()
...
-  
-  if (name_dir != "") {
+  std::string name_file;
+  std::string name_out_file  = expand_name_(&file_name_,&file_args_);
+
+  if (name_dir == "") {
+    // output block list and parameters to work directory
+    name_dir  = ".";
+    // strip extension, use this for name
+    name_file = name_out_file.substr(0, name_out_file.find("."));
```
Comment (brittonsmith) [inline src/Cello/io_OutputData.cpp:145]: Searching for the last period with `rfind` might be a bit more robust.

**Comment by aemerick** [comment]
This should be good to go!

**Comment by bwoshea** [comment]
Four approvals, merging!

**Pull request merged**